src/dialogs/more-info/controls/more-info-media_player.ts

```diff
@@ -214,6 +214,7 @@ class MoreInfoMediaPlayer extends LitElement {
       ha-icon-button[action="turn_off"],
       ha-icon-button[action="turn_on"] {
         margin-inline-end: auto;
+        margin-inline-start: inherit;
```

Comment:
Shouldn't these be below the left and right so they override them?

Reply:
My env is bad. I'm closing this one and submitting another PR later